apps/generic_terminal/unix_utilities.py

```diff
@@ -0,0 +1,90 @@
+from talon import Context, Module
+
+from ...code.user_settings import get_list_from_csv
+
+ctx = Context()
+mod = Module()
+ctx.matches = r"""
+tag: terminal
+and tag: user.unix_utilities
+"""
+
+mod.tag(
+    "unix_utilities", desc="tag for enabling unix utility commands in your terminal"
+)
+
+# Do not edit this dictionary. It is just used to initially populate 'settings/unix_utilities.csv'.
+# Edit this file instead.
+default_unix_utilities = {
```

Comment:
The rsync one is a bit difficult for my Australian accent, but I think on the whole this is a good translation to voice. It also has a lot of the utilities I use regularly (and that aren't in the existing voice commands). Nice one!